components/led_strip/led_strip.c

```diff
@@ -153,13 +153,13 @@ esp_err_t led_strip_init(led_strip_t *strip)
 {
     CHECK_ARG(strip && strip->length > 0);
 
-    strip->buf = malloc(strip->length * COLOR_SIZE);
+    strip->buf = malloc(strip->length * (COLOR_SIZE + strip->is_rgbw));
```

Comment:
This way of calculating buffer size doesn't seem correct to me, because it will not work in cases where `is_rgbw` is greater than 1.

Reply:
Well since is_rgbw is bool, I did not considered this case. 
But with [this](https://github.com/UncleRus/esp-idf-lib/pull/126/commits/19b5a597395b8bf6b8d152893b943d8615cc44f9), the case does not exist anymore.